load.php

```diff
@@ -59,7 +59,14 @@ function perflab_sanitize_modules_setting( $value ) {
  * @return array Associative array of module settings keyed by module slug.
  */
 function perflab_get_module_settings() {
-	return (array) get_option( PERFLAB_MODULES_SETTING, array() );
+	return (array) get_option(
+		PERFLAB_MODULES_SETTING,
+		array(
+			'webp-default' => array(
+				'enabled' => true,
+			),
+		)
+	);
 }
```

Comment:
Let's not change the default here - all modules should be disabled by default, at least for now. Users should opt in into which performance features they want to test rather than having to opt out.

Reply:
👍🏼  yep, this was temporary for testing, will remove.

Reply:
removed